app/controllers/v0/sign_in_controller.rb

```diff
@@ -103,10 +103,10 @@ def token
 
       StatsD.increment(SignIn::Constants::Statsd::STATSD_SIS_TOKEN_SUCCESS)
       render json: response_body, status: :ok
-    rescue SignIn::Errors::StandardError => e
+    rescue SignIn::Errors::StandardError, SignIn::Errors::CredentialLockedError => e
```

Comment:
I don't think you need any of the changes in this file anymore, rescuing the `SignIn::Errors::StandardError` will cover this sufficiently